cmd/influxd/launcher/cmd.go

```diff
@@ -70,19 +70,20 @@ func cmdRunE(ctx context.Context, o *InfluxdOpts) func() error {
 	return func() error {
 		fluxinit.FluxInit()
 
-		// exit with SIGINT and SIGTERM
-		ctx = signals.WithStandardSignals(ctx)
-
 		l := NewLauncher()
-		if err := l.run(ctx, o); err != nil {
+
+		// Start the launcher and wait for it to exit on SIGINT or SIGTERM.
+		runCtx := signals.WithStandardSignals(ctx)
```

Comment:
why assign this to `runCtx` and not simply:

```
ctx := signals.WithStandardSignals(ctx)
```


also, if you're using go 1.15, i think you can use `os/signal.NotifyContext()`.
?